runtime/src/main/java/org/corfudb/protocols/wireprotocol/orchestrator/OrchestratorMsg.java

```diff
@@ -41,6 +41,10 @@ static Request mapRequest(OrchestratorRequestType type, byte[] payload) {
         switch (type) {
             case ADD_NODE:
                 return new AddNodeRequest(payload);
+            case REMOVE_NODE:
+                return new RemoveNodeRequest(payload);
+            case FORCE_REMOVE:
```

Comment:
nit:is there a better name than FORCE_REMOVE ? It is not growing on me given that the other 2 I see are ADD_NODE and REMOVE_NODE. But as I said it is a minor nit.